// Copyright 2019 Google LLC. All Rights Reserved.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     http://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.
syntax = "proto3";

package tfx.components.tuner;

// Args specific to tuning.
message TuneArgs {
  // Number of trials to run in parallel. Each trial will be trained and
  // evaluated by separate worker jobs.
  // Note parallel trials are not supported in default in-process tuning.
  int32 num_parallel_trials = 1;

  reserved 2, 3, 4;
}
